cogs/poll.py

```diff
@@ -0,0 +1,66 @@
+import discord
+from discord.ext import commands
+
+
+class Poll(commands.Cog):
+    """"""
```

Comment:
Refactor and format all files properly

Reply:
yes I have a lot of cleaning up and documentation to do

Reply:
and imma get rid of the reddit stuff as well too thank you